extensions/BMO/template/en/default/bug/create/comment-ipc.txt.tmpl

```diff
@@ -0,0 +1,46 @@
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0.
+  #%]
+
+[% USE Bugzilla %]
+[% cgi = Bugzilla.cgi %]
+
+>> What team are you are a part of?
+[%+ cgi.param("team_name") %]
+
+>> What is the user story associated with your snippet?
+[%+ cgi.param("user_story") %]
+
```

Comment:
I wonder if the requester wants the user story field to be in comment zero, or if they want it in the user story field?